[2022-08-06 19:37:42,306] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-09T00:00:00+00:00 [queued]>
[2022-08-06 19:37:42,323] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-09T00:00:00+00:00 [queued]>
[2022-08-06 19:37:42,323] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:37:42,323] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:37:42,323] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:37:42,340] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-01-09 00:00:00+00:00
[2022-08-06 19:37:42,348] {standard_task_runner.py:52} INFO - Started process 1646 to run task
[2022-08-06 19:37:42,351] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-01-09T00:00:00+00:00', '--job-id', '494', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpwsfg612r', '--error-file', '/tmp/tmp9_yq0qhf']
[2022-08-06 19:37:42,351] {standard_task_runner.py:80} INFO - Job 494: Subtask second_task
[2022-08-06 19:37:42,425] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-09T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:37:42,524] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-09T00:00:00+00:00
[2022-08-06 19:37:42,525] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:37:42,525] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:37:42,537] {subprocess.py:85} INFO - Output:
[2022-08-06 19:37:42,539] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:37:42,540] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:37:42,571] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220109T000000, start_date=20220806T193742, end_date=20220806T193742
[2022-08-06 19:37:42,602] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:37:42,639] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:51:35,780] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-09T00:00:00+00:00 [queued]>
[2022-08-06 21:51:35,796] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-09T00:00:00+00:00 [queued]>
[2022-08-06 21:51:35,797] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:51:35,797] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:51:35,797] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:51:35,814] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-01-09 00:00:00+00:00
[2022-08-06 21:51:35,821] {standard_task_runner.py:52} INFO - Started process 1828 to run task
[2022-08-06 21:51:35,826] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-01-09T00:00:00+00:00', '--job-id', '493', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp0eaqv4mn', '--error-file', '/tmp/tmp4_zx93tc']
[2022-08-06 21:51:35,826] {standard_task_runner.py:80} INFO - Job 493: Subtask second_task
[2022-08-06 21:51:35,904] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-09T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:51:36,023] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-09T00:00:00+00:00
[2022-08-06 21:51:36,024] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:51:36,024] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:51:36,039] {subprocess.py:85} INFO - Output:
[2022-08-06 21:51:36,043] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:51:36,043] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:51:36,082] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220109T000000, start_date=20220806T215135, end_date=20220806T215136
[2022-08-06 21:51:36,116] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:51:36,156] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:57:00,287] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-09T00:00:00+00:00 [queued]>
[2022-08-06 21:57:00,306] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-09T00:00:00+00:00 [queued]>
[2022-08-06 21:57:00,306] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:57:00,306] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:57:00,306] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:57:00,333] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-01-09 00:00:00+00:00
[2022-08-06 21:57:00,342] {standard_task_runner.py:52} INFO - Started process 1643 to run task
[2022-08-06 21:57:00,346] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-01-09T00:00:00+00:00', '--job-id', '498', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpwc5339o4', '--error-file', '/tmp/tmp7i7ca8s6']
[2022-08-06 21:57:00,346] {standard_task_runner.py:80} INFO - Job 498: Subtask second_task
[2022-08-06 21:57:00,429] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-09T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:57:00,536] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-09T00:00:00+00:00
[2022-08-06 21:57:00,537] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:57:00,537] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:57:00,550] {subprocess.py:85} INFO - Output:
[2022-08-06 21:57:00,551] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:57:00,551] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:57:00,594] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220109T000000, start_date=20220806T215700, end_date=20220806T215700
[2022-08-06 21:57:00,639] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:57:00,686] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:12:01,736] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-09T00:00:00+00:00 [queued]>
[2022-08-06 22:12:01,749] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-09T00:00:00+00:00 [queued]>
[2022-08-06 22:12:01,749] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:12:01,749] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:12:01,749] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:12:01,765] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-01-09 00:00:00+00:00
[2022-08-06 22:12:01,772] {standard_task_runner.py:52} INFO - Started process 1639 to run task
[2022-08-06 22:12:01,775] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-01-09T00:00:00+00:00', '--job-id', '495', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp20dve9l8', '--error-file', '/tmp/tmp5wfce9zg']
[2022-08-06 22:12:01,775] {standard_task_runner.py:80} INFO - Job 495: Subtask second_task
[2022-08-06 22:12:01,833] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-09T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:12:01,902] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-09T00:00:00+00:00
[2022-08-06 22:12:01,903] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:12:01,903] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:12:01,915] {subprocess.py:85} INFO - Output:
[2022-08-06 22:12:01,917] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:12:01,917] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:12:01,941] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220109T000000, start_date=20220806T221201, end_date=20220806T221201
[2022-08-06 22:12:01,986] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:12:02,013] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:23:21,968] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-09T00:00:00+00:00 [queued]>
[2022-08-06 22:23:21,978] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-09T00:00:00+00:00 [queued]>
[2022-08-06 22:23:21,978] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:23:21,978] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:23:21,978] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:23:21,995] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-01-09 00:00:00+00:00
[2022-08-06 22:23:22,002] {standard_task_runner.py:52} INFO - Started process 1689 to run task
[2022-08-06 22:23:22,004] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-01-09T00:00:00+00:00', '--job-id', '497', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmps14n1vha', '--error-file', '/tmp/tmp524tnac_']
[2022-08-06 22:23:22,004] {standard_task_runner.py:80} INFO - Job 497: Subtask second_task
[2022-08-06 22:23:22,058] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-09T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:23:22,130] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-09T00:00:00+00:00
[2022-08-06 22:23:22,130] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:23:22,131] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:23:22,143] {subprocess.py:85} INFO - Output:
[2022-08-06 22:23:22,144] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:23:22,145] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:23:22,169] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220109T000000, start_date=20220806T222321, end_date=20220806T222322
[2022-08-06 22:23:22,215] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:23:22,240] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
